gnssanalysis/gn_io/sp3.py

```diff
@@ -508,9 +508,9 @@ def clk_std_formatter(x):
 
         # POS nodata formatting
         # Fill +/- infinity values with SP3 nodata value for POS columns
-        epoch_vals["X"].replace(to_replace=[_np.inf, _np.NINF], value=SP3_POS_NODATA_STRING, inplace=True)
-        epoch_vals["Y"].replace(to_replace=[_np.inf, _np.NINF], value=SP3_POS_NODATA_STRING, inplace=True)
-        epoch_vals["Z"].replace(to_replace=[_np.inf, _np.NINF], value=SP3_POS_NODATA_STRING, inplace=True)
+        epoch_vals["X"].replace(to_replace=[_np.inf, -_np.inf], value=SP3_POS_NODATA_STRING, inplace=True)
```

Comment:
This will need updating again for Pandas 3. I'll try to replace it with `format()` calls at the point of writing the data out.

Reply:
> This will need updating again for Pandas 3. I'll try to replace it with `format()` calls at the point of writing the data out.

Yeah, there's a few changes that we'll need to make in order to make us Pandas 3 compliant. Running the `sp3merge` utility already has a whole bunch of warnings around this